learn/tanssi/staking.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Staking
+description: Learn how Tanssi implements a novel Staking mechanism to provide a decentralized and trustless set of block producers to all Tanssi-powered Appchains
+---
+
+# Tanssi Staking {: #tanssi-staking }
+
+## Introduction {: #introduction }
+
+One of Tanssi's core propositions is to simplify the infrastructure complexity for Appchains. A significant component is bootstrapping a decentralized set of block producers, which Tanssi offers through its unique architecture and staking mechanics.
+
+Tanssi staking mechanics guarantee that the block producers for Tanssi and Tanssi-powered Appchains are selected through a trustless and decentralized mechanism. They also incentivize the community to delegate to top-performing or engaged block producers.
+
+This page covers the fundamental concepts of Tanssi's staking mechanics and how it secures a decentralized block production set that drives network liveness for Tanssi Appchains.
+
+## Core Concepts {: #core-concepts }
+
+Tanssi's staking module mechanics were inspired by the concept of liquidity pool tokens (LP tokens) in traditional Automated-Market-Makers (AMMs) like Uniswap V2.
+
+Each block producer has four liquidity pools through which delegators move as they perform different staking operations. In short, each liquidity pool represents a different state throughout the staking process: joining, staking through manual rewards, staking through auto-compound rewards, and leaving. Nevertheless, one core difference is that LP tokens in common AMMs are transferable while staking shares tokens are not.
+
+A delegator has four simple transactions to go through the different states (liquidity pools): delegate (for manual or auto-compound rewards), undelegate, swap, and execute pending operations. For example, users who want to stake through either rewards pool can use the `delegate` call and join the Joining Pool immediately. After a delay, users (or anyone else) can `execute pending operation` and enter the initially set rewards pool. Once there, users can `swap` between reward pools as often as they like. Lastly, users in a rewards pool can use the `undelegate` call to go into the Leaving Pool and unstake their tokens (or anyone else's) `executing pending operation` after a given delay.
```

Comment:
I think the use of back ticks here is a bit out of the norm. If referring to a specific function, it should be wrapped in back ticks, and the spelling should be the same as the function name, i.e., `executePendingOperations`. If you're just saying that anyone can execute the pending operations, then I wouldn't wrap it in back ticks. 

Since we're not actually defining the exact extrinsics here, I just wouldn't use back ticks

Reply:
sounds good